raddb/mods-config/perl/oauth2.pm

```diff
@@ -0,0 +1,400 @@
+use strict;
```

Comment:
We need to state (c) ownership on that file + license.

Reply:
Just to make sure, would adding the following perldoc work:

```
= head1 COPYRIGHT

Copyright Alexander Clouter (https://github.com/jimdigriz)

Licence: https://github.com/jimdigriz/freeradius-oauth2-perl/blob/master/LICENSE
```

Reply:
Yes it should be OK.